library/block/block_content_registry/src/main/java/org/quiltmc/qsl/block/content/registry/api/BlockContentRegistries.java

```diff
@@ -34,64 +34,63 @@
  * <p>
  * Current properties:
  * <ul>
- *     <li>{@link #FLATTENABLE_BLOCK}</li>
- *     <li>{@link #OXIDIZABLE_BLOCK}</li>
- *     <li>{@link #WAXABLE_BLOCK}</li>
- *     <li>{@link #STRIPPABLE_BLOCK}</li>
- *     <li>{@link #FLAMMABLE_BLOCK}</li>
- *     <li>{@link #SCULK_FREQUENCY}</li>
+ *     <li>{@link #FLATTENABLE_BLOCKS}</li>
+ *     <li>{@link #OXIDIZABLE_BLOCKS}</li>
+ *     <li>{@link #WAXABLE_BLOCKS}</li>
+ *     <li>{@link #STRIPPABLE_BLOCKS}</li>
+ *     <li>{@link #FLAMMABLE_BLOCKS}</li>
  * </ul>
  */
 public class BlockContentRegistries {
 	/**
 	 * The namespace for the content registries.
 	 */
-	public static final String NAMESPACE = "quilt_block_content_registry";
+	public static final String NAMESPACE = "quilt";
 
 	/**
 	 * A {@link RegistryEntryAttachment} for when blocks are right clicked by a shovel.
 	 * <p>
-	 * Values can be set via code and through a data-pack with the file {@code data/quilt_block_content_registry/attachments/minecraft/block/flattenable_block.json}
+	 * Values can be set via code and through a data-pack with the file {@code data/quilt/attachments/minecraft/block/flattenable_blocks.json}
 	 */
-	public static final RegistryEntryAttachment<Block, BlockState> FLATTENABLE_BLOCK = RegistryEntryAttachment
+	public static final RegistryEntryAttachment<Block, BlockState> FLATTENABLE_BLOCKS = RegistryEntryAttachment
 			.builder(Registry.BLOCK,
-					new Identifier(NAMESPACE, "flattenable_block"),
+					new Identifier(NAMESPACE, "flattenable_blocks"),
```

Comment:
Hmm, we'll probably want `quilt:flattenable` than the explicit `_blocks` suffix there; This applies to the rest of the BCR

Reply:
Ah yep. I'll fix this.

Reply:
making the item REAs plural does make sense, but i'm even more conflicted at `flattenables`; Remember, the goal of renaming the REAs is to make their naming scheme fit tags, 'cause well, they are supposed to be KV tags after all

Reply:
I'm not sure every one of this are correct English :p

Flammables is correct, flattenable isn't.

Reply:
-able is a living suffix that is valid to put on any verb. it is correct English even if the word isn't used enough to warrant going in a dictionary

Reply:
> -able is a living suffix that is valid to put on any verb. it is correct English even if the word isn't used enough to warrant going in a dictionary

Well, I've looked up flattenable and on Wiktionary it straight up doesn't have a plural form, while flammables have.
So uh, *shrugs*

Reply:
Sounds like the `-able` should not have the `-s`